prow/cmd/deck/job_history.go

```diff
@@ -363,6 +365,16 @@ func getBuildData(ctx context.Context, bucket storageBucket, dir string) (buildD
 		logrus.Debugf("failed to read finished.json (job might be unfinished): %v", err)
 	}
 
+	pj := prowapi.ProwJob{}
+	err = readJSON(ctx, bucket, path.Join(dir, "prowjob.json"), &pj)
```

Comment:
I think `prowjob.json` should be turned into a constant that both this and the producer uses.